blocksuite/framework/std/src/gfx/interactivity/types/view.ts

```
@@ -79,8 +183,10 @@ export type GfxViewTransformInterface = {
   onDragStart: (context: DragStartContext) => void;
   onDragMove: (context: DragMoveContext) => void;
   onDragEnd: (context: DragEndContext) => void;
-  onRotate: (context: {}) => void;
-  onResize: (context: {}) => void;
+
+  onRotateStart: (context: {}) => void;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)